### DISCUSSION TITLE
Aiuto configurazione luci rgbw Local Tuya

### DISCUSSION THREAD

**Discussion by blakdog1978** [indomus/forum#2124] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
Buongiorno, stavo provando a configurare delle luci rgbw in local tuya ma non riesco a capire che parametri devo mettere...
Nella prima immagine sono i parametri da configurare e nella seconda le varie scelte possibili
![local tuya](https://user-images.githubusercontent.com/36457031/204475152-52e02d75-b709-411f-82a7-c5e70b789779.JPG)
![local tuya 2](https://user-images.githubusercontent.com/36457031/204475189-7ca99581-7b5a-4a7a-8bbc-9ad4d5b88e9b.JPG)

**Comment by brisicche78** [comment]
anche io ho le stesso problema sono solo riuscito a farla accendere in modalita classica perdendo tutte le impostazioni dei colori. spero qualche buon anima ci possa aiutare

**Comment by blakdog1978** [comment]
Io ho risolto leggendo qui:
https://github.com/rospogrigio/localtuya/issues/587

**Reply by brisicche78** [reply]
non riesco a capire come fare puoi aiutarmi ho aperto il link che hai mandato ma non capisco nulla help me please

**Comment by blakdog1978** [comment]
Ciao non so che DPS ti dia da scegliere nel mio caso erano 20-21-22-23-24-25 e seguendo questo

![dp](https://user-images.githubusercontent.com/36457031/206165183-8265bb64-3224-4798-ab81-a183e733ce07.JPG)

ho impostato le mie lampadine così

![dp impostati](https://user-images.githubusercontent.com/36457031/206165425-a2317730-5635-4862-ac9d-db8eea49907e.JPG)

Poi per le scene vedrai che non corrispondono a quelle che hai impostato nell'app tuya ma saranno quelle che ci sono configurate nel file light.py e le potrai configurare li....
